tests/functional/database-management.test.js

```diff
@@ -9,13 +9,107 @@ describe('database management', function () {
 
     context('database creation and deletion', function () {
 
+        (harnessConfigs.testRemoteSwarm ? context.skip : context)('permissioning', function () {
+
+            const randomUuid = `${Math.random()}`;
+
+            const notMasterKeyPair = {
+                privateKey: 'MHQCAQEEIKyRad3bhvLOMC9/zajsk5+o9WIaQoWNZMPjN+RauDOSoAcGBSuBBAAKoUQDQgAEs/FPun/4jYE+vitiFOGxo/Wxy1Zsv1UjqDwVfnI45qehmPBd7VxvPyX9fHbwtFUZFp8S5+9B/gwBvKN/2+5R6g==',
+                publicKey: 'MFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAEs/FPun/4jYE+vitiFOGxo/Wxy1Zsv1UjqDwVfnI45qehmPBd7VxvPyX9fHbwtFUZFp8S5+9B/gwBvKN/2+5R6g=='
+            };
+
+            (harnessConfigs.testRemoteSwarm
+                ? remoteSwarmHook({
+                    createDB: false,
+                    uuid: randomUuid,
+                    private_pem: harnessConfigs.masterPrivateKey,
+                    public_pem: harnessConfigs.masterPublicKey
+                })
+                : localSwarmHooks({
+                    createDB: false,
+                    configOptions: {owner_public_key: harnessConfigs.masterPublicKey},
+                    private_pem: harnessConfigs.masterPrivateKey,
+                    public_pem: harnessConfigs.masterPublicKey
+                }));
+
+            context('with matching client private key and daemon owner_public_key', function () {
+
+                it('should be able to successfully createDB', async function () {
+                    await this.api._createDB()
```

Comment:
Maybe throw in an assertion before for `!hasDB`, and `hasDB` after.

Reply:
Done